### DISCUSSION TITLE
Use the script for third party javascript

### DISCUSSION THREAD

**Discussion by ghost** [mirceapiturca/Sections#8] [Q&A] [answered]
Hi Mircea,

I want to use the same logic to load script based on user interaction for third-party javascript and other javascript files that are available in /assets folder. How can I achieve that?

**Comment by mirceapiturca** [comment, marked as answer]
@Ghaleb-alnakhlani It would be a good feature to have.
You can create your own custom loading logic by using some of the section code:
https://github.com/mirceapiturca/Sections/blob/master/App%20Optimization/app-optimization.liquid#L122

`node.addEventListener(interactionEvent.event, eventFn, eventOptions);`
Node is the DOM element mathed by the assigned CSS selectors.
interactionEvent.event is the assigned event.
eventFn is the function that loads the secipt, see: `loadScript(src)`

**Reply by ghost** [reply]
Thank you @mirceapiturca for the quick answer. If you don`t mind can you please show me an example of how to implement load on user interaction e.g scroll, on one of third party scripts? something that can be placed in theme.liquid not necessary using the app-optimization.liquid
